src/openklant/conf/api.py

```diff
@@ -3,6 +3,8 @@
 
 # Remove the reference - we don't have a single API version.
 del API_VERSION  # noqa
+# SITE_DOMAIN should be taken from env and not from commonground-api-common
+del SITE_DOMAIN  # noqa
```

Comment:
If you remove this line, will it take the default defined in `vng_api_common/api.py`? If that's the case it might be better to remove the default there, explicitly state in the docs of the library that `SITE_DOMAIN` needs to be defined in the settings.

I'd prefer that over having to delete things like this